src/components/Accordion/AccordionContent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import cn from 'classnames';
+
+import classes from './AccordionContent.module.css';
+import { useAccordionContext } from './Context';
+
+export interface AccordionContentProps {
+  children?: React.ReactNode;
+}
+
+export const AccordionContent = ({ children }: AccordionContentProps) => {
+  const { open } = useAccordionContext();
+
+  return (
+    <div>
+      {open && (
+        <div
+          className={cn(classes['accordion-content'], {
+            [classes['accordion-content--opened']]: open,
+          })}
```

Comment:
These classes are not used (after removing the `accordion-content`), and could be removed

```suggestion
```